contracts/tg4-stake/src/contract.rs

```diff
@@ -145,9 +146,14 @@ pub fn execute_bond<Q: CustomQuery>(
     deps: DepsMut<Q>,
     env: Env,
     info: MessageInfo,
+    vesting_tokens: Option<Coin>,
 ) -> Result<Response, ContractError> {
     let cfg = CONFIG.load(deps.storage)?;
     let amount = validate_funds(&info.funds, &cfg.denom)?;
+    let vesting_amount = vesting_tokens
```

Comment:
The `validate_funds()` fails when only vesting tokens were declared and liquid funds are empty.

Reply:
Will change it to a function that validates both at the same time, and checks that at least one of them is empty.